pipelines/toast_ground_schedule.py

```diff
@@ -1392,6 +1424,13 @@ def add_scan(
             subscan,
         )
         entries.append(entry)
+        #Accumulate the count for the number of scans of this elevation and drift.
+        #The += 1 could be changed if desired; for example, to amount of time spent, or something, instead of 1.
+        if args.elevations is not None:
+            if rising:
+                patch.el_count[[float(x) for x in args.elevations.split(',')].index(el/degree)][0] += 1;
```

Comment:
Searching for an index of a floating point number failed in my test with
```
  File "/Users/reijo/Software/macports/bin/toast_ground_schedule.py", line 1433, in add_scan
    patch.el_count[[float(x) for x in args.elevations.split(',')].index(el/degree)][1] += 1;
ValueError: 59.99999999999999 is not in list
```
The list included `60.0`. I suggest storing and comparing the elevations with some appropriate rounding, such as `np.round(float(x), 3)`. Furthermore, rather than generating the list of elevations repeatedly, I would make `patch.el_count` a Python dictionary.